src/authenticatedFetch/dpop/DpopAuthenticatedFetcher.ts

```diff
@@ -32,6 +32,17 @@ import { IDpopHeaderCreator } from "../../dpop/DpopHeaderCreator";
 import { IUrlRepresentationConverter } from "../../util/UrlRepresenationConverter";
 import { IStorageUtility } from "../../localStorage/StorageUtility";
 
+function flattenHeaders(
+  headersToFlatten: Headers | string[][] | Record<string, string> | undefined
```

Comment:
I don't think `string[][]` can occur.

Reply:
This is part of the type signature of `RequestInit.headers`.

Reply:
Weird, not on MDN. The `Headers` constructor takes it, but not the `headers` field as far as I can see. No biggy though, it will work.

Reply:
This issue is similar to the one discussed above: there's a divergence in the `Headers` constructors. This will be updated when we straighten out the typing.